File: 30DaysOfCode/day26_nestedLogic/nestedLogic.cpp
#include <cmath>
#include <cstdio>
#include <vector>
#include <iostream>
#include <algorithm>

using namespace std;


int main() {
    int dayReturned, monthReturned, yearReturned;
    int dayExpected, monthExpected, yearExpected;
    
    cin >> dayReturned >> monthReturned >> yearReturned;
    cin >> dayExpected >> monthExpected >> yearExpected;
    
    if (yearReturned > yearExpected) {
        cout << 10000 << endl;
    } else if (yearReturned == yearExpected) {
        if (monthReturned > monthExpected) {
            int fine = 500 * (monthReturned - monthExpected);
            cout << fine << endl;
        } else if (monthReturned == monthExpected) {
            if (dayReturned > dayExpected) {
                int fine = 15 * (dayReturned - dayExpected);
                cout << fine << endl;
            } else {
                cout << 0 << endl;
            }
        } else {
            cout << 0 << endl;
        }
    } else {
        cout << 0 << endl;
    }
    
    return 0;
}

File: 30DaysOfCode/day26_nestedLogic/nestedLogic.java
import java.io.*;
import java.util.*;

public class Solution {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int dayReturned = sc.nextInt();
        int monthReturned = sc.nextInt();
        int yearReturned = sc.nextInt();
        int dayExpected = sc.nextInt();
        int monthExpected = sc.nextInt();
        int yearExpected = sc.nextInt();
        
        if (yearReturned > yearExpected) {
            System.out.println(10000);
        } else if (yearReturned == yearExpected) {
            if (monthReturned > monthExpected) {
                int fine = 500 * (monthReturned - monthExpected);
                System.out.println(fine);
            } else if (monthReturned == monthExpected) {
                if (dayReturned > dayExpected) {
                    int fine = 15 * (dayReturned - dayExpected);
                    System.out.println(fine);
                } else {
                    System.out.println(0);
                }
            } else {
                System.out.println(0);
            }
        } else {
            System.out.println(0);
        }
    }
}
